stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsDialogFragment.kt

```diff
@@ -62,15 +62,15 @@ internal class MessageOptionsDialogFragment : FullScreenDialogFragment() {
     private val optionsMode: OptionsMode by lazy {
         requireArguments().getSerializable(ARG_OPTIONS_MODE) as OptionsMode
     }
+    private val reactionsEnabled: Boolean by lazy {
+        requireArguments().getBoolean(ARG_REACTIONS_ENABLED)
```

Comment:
same here, do we need the reacionsEnabled by itself? Maybe just extract it from the style iteslf?